### PR TITLE
Update Software Installation

### PR DESCRIPTION
**Pull request by neilrees**
Just gone through the first time setup, here's a few amendments to areas that have changed since the docs were written.

* Updated the use python 3.7 
* Use sudo for the call to `make install`

### CODE REVIEW

**Comment by ngardiner** [comment]
Thanks for that @neilrees. Which OS/version are you running on? With the recent deprecation of python2 this makes sense, I'll just need to cross-check how it affects those on an older distribution that still has a python2 interpreter (such as mine).

**Comment by neilrees** [comment]
That was on Raspbian Buster Lite 2020-02-13.

File: docs/modules/Control_MQTT.md [hunk 1]
```diff
@@ -42,6 +42,6 @@ The following table shows the available configuration parameters for the MQTT Co
 
 | MQTT Topic                     | Value | Description |
 | ------------------------------ | ----- | ----------- |
-| *prefix*/control/chargeNow     | TBA | Starts charging immediately. |
-| *prefix*/control/chargeNowEnd  | TBA | Stops immediate charging. |
+| *prefix*/control/chargeNow     | AA,SS | Starts charging at AA amps for SS seconds immediately. e.g. `8,3600` = 8 Amps for 1 hour |
```
Comment (neilrees) [inline docs/modules/Control_MQTT.md:45]: Is the second parameter seconds or is it minutes?

**Comment by nean-and-i** [comment]
Hi guys,

in addition to the `make` approach, would using virtualenv and deploying the framework via pip make sense to ensure integrity, consistency and that the python dependencies are met ?

something like:

```bash
mkdir /usr/local/twcamanager
cd /usr/local/twcmanager
virtualenv -ppython3 .
source bin/activate
pip install twcmanager-1.3.tar.gz
```

/usr/local/twcamanager/etc -> location for config.json and twcmanager.service (rather than /etc/twcmanager)
/usr/local/twcamanager/bin -> TWCManager.py
etc,...

this could then be a nice portable package ?

thanks

File: docs/modules/Control_MQTT.md (hunk 1 above)
Comment (ngardiner) [reply docs/modules/Control_MQTT.md:45]: Apologies for the late reply, @neilrees. It is indeed seconds as you have in the table above. Thanks for the documentation improvement, it is very useful!

**Comment by ngardiner** [comment]
Thanks for the PR, @neilrees! I am going to merge this in two parts.

- MQTT chargeNow documentation and output I will merge now.
- The updates to the README/install instructions I will hold off for now. The reasons are:

- I haven't had an opportunity yet to look at which raspbian versions will require which packages. The concern being that commands which work for one person may not work for another based on when they installed the system.

- 1.2.0 is the current development branch, whilst 1.1.8 is the current stable branch. It's just a slight update needed to the PR, to suggest that those who want a stable install should use 1.1.8 and those who want the newest features (at the risk of occasional breakage) can use 1.2.0.

**Comment by ngardiner** [comment]
Thanks @neilrees, I have merged your documentation update and I'll close this PR off now.

@nean-and-i, thanks for the suggestion and apologies you didn't get a reply here, I will move your suggestion to a separate issue and close off this PR, I think it would be nice to have a portable virtualenv as you've described and perhaps we could add it to the Makefile. Let's follow it up in an issue dedicated to the suggestion.

**Pull request closed without merging**